crypto/bn/bn_ppc.c

```diff
@@ -40,12 +40,14 @@ int bn_mul_mont(BN_ULONG *rp, const BN_ULONG *ap, const BN_ULONG *bp,
      * no opportunity to figure it out...
      */
 
+#if _ARCH_PPC64
```

Comment:
Is this universally available? Should it be `#ifdef _ARCH_PPC64` instead?

Reply:
Good catch.  Thanks...

Other places use `#if defined(_ARCH_PPC64)` so I'll use that for consistency.